python/py-onnx/Portfile

```diff
@@ -69,11 +72,23 @@ if {${name} ne ${subport}} {
 
     build.env-append \
         pybind11_DIR=${prefix}/share/cmake/pybind11
+
     build.dir       ${worksrcpath}
     build.post_args
 
-    destroot.dir    ${worksrcpath}
+    pre-destroot {
+        destroot.args-append \
+                    [glob ${workpath}/${python.rootname}-${version}-*.whl]
+    }
+
+    destroot.cmd    pip-${python.branch}
     destroot.target install
+    destroot.args          \
+        --ignore-installed \
+        --no-cache-dir     \
+        --no-dependencies  \
+        --root ${destroot}
+    destroot.post_args
```

Comment:
same comment as in the other PR ; the "normal" PEP517 build should work here I would think. What's the error you are trying to workaround here?